File: src/main/java/me/kuku/yuq/service/MotionService.java
package me.kuku.yuq.service;

import com.IceCreamQAQ.Yu.annotation.AutoBind;
import me.kuku.yuq.entity.StepEntity;

import java.util.List;

@AutoBind
public interface MotionService {
    StepEntity findByQQ(long qq);
    List<StepEntity> findAll();
    void save(StepEntity stepEntity);
    void delByQQ(Long qq);
}

File: src/main/kotlin/me/kuku/yuq/service/MotionService.kt
package me.kuku.yuq.service

import com.IceCreamQAQ.Yu.annotation.AutoBind
import me.kuku.yuq.entity.MotionEntity

@AutoBind
interface MotionService {
    fun findByQQ(qq: Long): MotionEntity?
    fun findAll(): List<MotionEntity>
    fun save(motionEntity: MotionEntity)
    fun delByQQ(qq: Long)
}